src/main/resources/External_Resources.csv

```diff
@@ -0,0 +1,115 @@
+AddLinks,add-links/Main.java,add-links/src/main/java/addlinks/ensembl/EnsemblReferenceDatabaseGenerator.java#L128,EnsEMBL transcript summary page (mus musculus transcript),EnsEMBL Access URL for mouse transcript,Web Page,http://www.ensembl.org/mus_musculus/transview?transcript=ENSMUST00000207604
+AddLinks,add-links/Main.java,add-links/src/main/java/addlinks/ensembl/EnsemblReferenceDatabaseGenerator.java#L128,EnsEMBL peptide summary page (gallus gallus peptide),EnsEMBL Access URL for chicken peptide,Web Page,http://apr2005.archive.ensembl.org/gallus_gallus/protview?peptide=ENSGALP00000000406
+AddLinks,add-links/Main.java,add-links/src/main/java/addlinks/ensembl/EnsemblReferenceDatabaseGenerator.java#L128,EnsEMBL peptide summary page (rattus norvegicus peptide),EnsEMBL Access URL for rat peptide,Web Page,http://www.ensembl.org/rattus_norvegicus/protview?peptide=ENSRNOP00000006671
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L83,Monarch gene summary,Monarch Access URL,Web Page,https://monarchinitiative.org/gene/NCBIGene:3845
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L98,NCBI Gene in identifiers.org,NCBI Gene Access URL,Web Page,https://www.ebi.ac.uk/miriam/main/resources/MIR:00100099
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L112,FlyBase in identifiers.org,FlyBase Access URL,Web Page,https://www.ebi.ac.uk/miriam/main/resources/MIR:00100050
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L121,GeneCards in identifiers.org,GeneCards Access URL,Web Page,https://www.ebi.ac.uk/miriam/main/resources/MIR:00100407
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L145,HapMap in identifiers.org,HapMap Access URL,Web Page,https://www.ebi.ac.uk/miriam/main/resources/MIR:00100374
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L159,Complex Portal in identifiers.org,Complex Portal Access URL,Web Page,https://www.ebi.ac.uk/miriam/main/resources/MIR:00100875
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L173,IntEnz in identifiers.org,IntEnz Access URL,Web Page,https://www.ebi.ac.uk/miriam/main/resources/MIR:00100001
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L186,KEGG Gene gene summary page (human gene),KEGG Gene Access URL for human,Web Page,https://www.genome.jp/dbget-bin/www_bget?hsa:448835
+AddLinks,add-links/Main.java,add-links/src/main/resources/reference-databases.xml#L186,KEGG Gene gene summary page (bos taurus gene),KEGG Gene Access URL for cow,Web Page,https://www.genome.jp/dbget-bin/www_bget?bta:100048947

```

Comment:
Typically, if KEGG is down then anything beginning with `https://www.genome.jp/dbget-bin/www_bget?` will fail. Most of these species-specific URLs should not need to be tested individually. Unless you've experienced situations where some species work and some do not? Also, these all reference the same line in the reference-databases.xml file. This seems weird. AddLinks generates many KEGG ReferenceDatabases on the fly (possibly some of the ones listed here), so if some of these are dynamically generated, should they all be here.

Also, line 186 is for OMIM: https://github.com/reactome/add-links/blob/master/src/main/resources/reference-databases.xml#L186

Reply:
I generated the list of resources many months ago before starting to code the project.  I'm using these for testing as I develop, but I've already found some that are out of date.  I'm going to be updating the resource list as I go.

Reply:
Created issue #4